set.mm

```diff
@@ -51029,15 +51037,33 @@ That theorem bundles the theorems ( ` |- E. x ( x = y -> z e. x ) ` with
   $}
 
   ${
-    $d x y $.
-    $( Alternate proof of ~ dtru using ~ ax-pr instead of ~ ax-pow .
-       (Contributed by Mario Carneiro, 31-Aug-2015.)
-       (Proof modification is discouraged.)  (New usage is discouraged.) $)
-    dtruALT2 $p |- -. A. x x = y $=
-      ( cv wceq wn wex wal c0 csn 0inp0 snex eqeq2 notbid spcev syl 0ex pm2.61i
-      exnal eqcom albii xchbinx mpbi ) BCZACZDZEZAFZUDUCDZAGZEUCHDZUGUJUCHIZDZE
-      ZUGUCJUFUMAUKHKUDUKDUEULUDUKUCLMNOUFUJEAHPUDHDUEUJUDHUCLMNQUGUEAGUIUEARUE
-      UHAUCUDSTUAUB $.
+    $d w x y z $.
+    $( At least two sets exist (or in terms of first-order logic, the universe
+       of discourse has two or more objects).  Note that we may not substitute
+       the same variable for both ` x ` and ` y ` (as indicated by the distinct
+       variable requirement), for otherwise we would contradict ~ stdpc6 .
+
+       This theorem is proved directly from set theory axioms (no set theory
+       definitions) and does not use ~ ax-ext , ~ ax-sep , or ~ ax-pow .  See
+       ~ dtruALT for a shorter proof using these axioms, and see ~ dtruALT2 for
+       a proof that uses ~ ax-pow instead of ~ ax-pr .
+
+       The proof makes use of dummy variables ` z ` and ` w ` which do not
+       appear in the final theorem.  They must be distinct from each other and
+       from ` x ` and ` y ` .  In other words, if we were to substitute ` x `
+       for ` z ` throughout the proof, the proof would fail.  (Contributed by
+       NM, 7-Nov-2006.)  Avoid ~ ax-13 .  (Revised by Gino Giotto, 5-Sep-2023.)
+       Avoid ~ ax-12 .  (Revised by Rohan Ridenour, 9-Oct-2024.)  Avoid
+       ~ ax-pow .  (Revised by BTernaryTau, 3-Dec-2024.) $)
```

Comment:
This seems to remove ax-ext, ax-sep, and not ax-pow  from the proof of dftruALT2.  So the revision comment should be sth like "Base dftru on ax-pr, and avoid ax-pow instead."

Reply:
Updated.